modules/fedimint-ln-client/src/lib.rs

```diff
@@ -468,19 +551,19 @@ impl ClientModule for LightningClientModule {
 }
 
 #[derive(Error, Debug, Serialize, Deserialize)]
-pub enum LightningPayError {
+pub enum PayError {
     #[error("Lightning payment was canceled")]
```

Comment:
Should we remove "Lightning" from all of these error messages or is `Failed` the only one that is re-used in the internal payment case?

Reply:
Reverted the string change in `Failed` error variant. `PayError` is no-longer shared between the two, and I think we should refactor to get rid of `await_lightning_payment_success` and `await_refund` functions. There map error cases into `PayError`, but all of these errors are then swallowed when yielding extension state, for example in L398.